packages/appstore/src/components/options/options.scss

```diff
@@ -0,0 +1,180 @@
+.options-container,
```

Comment:
change the name of the file to options-account.scss

Reply:
The folder name and .tsx file names are the same.